### PR TITLE
add :Cart management

### PR DESCRIPTION
**Pull request by kavyakunder**
cart management

my-cart

### CODE REVIEW

**Review by shuklahimanshu81 — APPROVED** [review]
Code structure is neat. Good to merge from my side.

File: pages/cart-management.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/gh/hung1001/font-awesome-pro@4cac1a6/css/all.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="https://robocon-ui.netlify.app/styles.css" />
+    <link rel="stylesheet" href="../css/cart-management.css" />
+    <title>Document</title>
```
Comment (dev-enforced) [inline pages/cart-management.html]: Renaming it to cart

File: css/cart-management.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.cart-img {
+  width: 12rem;
+  height: 12rem;
+}
+
+.carts-total {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var(--light-grey-border);
+  border-radius: 10px;
```
Comment (dev-enforced) [inline css/cart-management.css]: If 10px is used repeatedly then you can use it as a common variable for easy accessibility

**Review by dev-enforced — APPROVED** [review]
The code looks good to me you can merge it!!Some minor suggestions which you can take care of later:

**Review by Shivanshsharma01 — APPROVED** [review]
Clean & well-structured code. Good to merge

File: pages/cart-management.html (hunk 1 above)
Comment (kavyakunder) [reply pages/cart-management.html]: done!

File: css/cart-management.css (hunk 2 above)
Comment (kavyakunder) [reply css/cart-management.css]: done!

**Pull request merged**